packages/components/package.json

```diff
@@ -71,7 +70,9 @@
     "@react-stately/select": "^3.5.5",
     "@react-types/shared": "^3.21.0",
     "classnames": "^2.3.2",
+    "color-string": "^1.9.1",
```

Comment:
This appears only used in `design-tokens` and some global docs (therefore goes on the root as a devDep)